bin/main.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/ruby
+#
+require_relative '../lib/tictactoe'
+
+class MainIO
```

Comment:
What does "Main" mean? Does a more descriptive name exist?

Reply:
Are there any tests for this?

Reply:
Does this belong in `main.rb`?

Reply:
I second all of these in suggestion form:
- `MainIO` doesn't strike me as a bad name, but based on my understanding of what this class is doing, I can think of a few names that I would find descriptive of the class's role, such as `UserInterface` or something like that. One thing I perceive `MainIO` doing as a name that I find less helpful generally speaking is pointing to implementation-specific details - namely that is is used in `main.rb` and wraps the standard IO globals.
- I think this class can be tested, and I think you're already set up nicely for it due to how you're injecting its dependencies, stdin and stdout.
- I think this class can be pulled into a separate file! Why do this? If the class is only used in one place (here) then I think the only argument is ease of testing (you can import the class under test and not additional unrelated things) and for better readability. Also if you have a 1-1 relationship between classes and files, the file tree of your project can be a useful representation of the class architecture of your program.